godot-core/src/builtin/quaternion.rs

```diff
@@ -0,0 +1,288 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+use std::ops::*;
+
+use godot_ffi as sys;
+use sys::{ffi_methods, GodotFfi};
+
+use crate::builtin::{math::*, vector3::*};
```

Comment:
```rs
use crate::builtin::*;
```
should also work, as both `math` and `vector3` types are re-exported in the `builtin` module.

Reply:
this is true, i had it like the above to be more specific to what i was importing. fixed